### PR TITLE
feat: DVM2.0 Add sync method for internal account slashing trackers

### PR DESCRIPTION
**Pull request by chrismaree**
**Motivation**

There are situations where we'd like to be able to partially sync account slashing trackers. this PR adds the ability to do this.

**Testing**

Check a box to describe how you tested these changes and list the steps for reviewers to test.

- [ ]  Ran end-to-end test, running the code as in production
- [ ]  New unit tests created
- [ ]  Existing tests adequate, no new tests required
- [ ]  All existing tests pass
- [ ]  Untested

**Issue(s)**

Fixes #XXXX

### CODE REVIEW

File: packages/core/contracts/oracle/implementation/VotingV2.sol [hunk 1]
```diff
@@ -332,20 +347,21 @@ contract VotingV2 is
 
             VoteInstance storage voteInstance = priceRequest.voteInstances[priceRequest.lastVotingRound];
 
-            uint256 totalVotes = voteInstance.resultComputation.totalVotes;
-            uint256 totalCorrectVotes = voteInstance.resultComputation.getTotalCorrectlyVotedTokens();
```
Comment (mrice32) [inline packages/core/contracts/oracle/implementation/VotingV2.sol]: Stack too deep?

File: packages/core/contracts/oracle/implementation/VotingV2.sol [hunk 2]
```diff
@@ -300,14 +320,9 @@ contract VotingV2 is
...
         // not. Based on the outcome, attribute the associated slash to the voter.
         int256 slash = 0;
-        for (
-            uint256 requestIndex = voterStake.lastRequestIndexConsidered;
-            requestIndex < priceRequestIds.length;
-            requestIndex = unsafe_inc(requestIndex)
-        ) {
+        for (uint256 requestIndex = indexFrom; requestIndex < indexTo; requestIndex = unsafe_inc(requestIndex)) {
             if (deletedRequests[requestIndex] != 0) requestIndex = deletedRequests[requestIndex] + 1;
             if (requestIndex > priceRequestIds.length - 1) break; // This happens if the last element was a rolled vote.
```
Comment (mrice32) [inline packages/core/contracts/oracle/implementation/VotingV2.sol:325]: Do we need to change this to `requestIndex > indexTo - 1` or anything?

**Review by mrice32 — APPROVED** [review]
This is great! Way simpler implementation than I was expecting +1

File: packages/core/contracts/oracle/implementation/VotingV2.sol (hunk 1 above)
Comment (chrismaree) [reply packages/core/contracts/oracle/implementation/VotingV2.sol]: yup

**Pull request merged**